res/mac/FreenetTray/src/defaults.plist

```diff
@@ -2,8 +2,12 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>firstlaunch</key>
+	<true/>
 	<key>nodepath</key>
 	<string>$INSTALL_PATH</string>
+	<key>nodefcpurl</key>
+	<string>tcp://127.0.0.1:9481</string>
```

Comment:
You're right that hardcoding of those ports should be parsed by the installer if there's a variable for it.

There should always be an `lo0` interface up, and the default `/etc/hosts` file points `localhost` to `lo0`'s `127.0.0.1` for IPv4 and `::1` for IPv6, so it should definitely be safe to use.